toplevel/ccompile.ml

```diff
@@ -112,6 +112,7 @@ let compile_file opts stm_opts copts injections (f_in, echo) =
   if !Flags.beautify then
     Flags.with_option Flags.beautify_file
```

Comment:
That's a no-op right?

Reply:
I don't know what the difference is between beautify and beautify_file